packages/esm-patient-registration-app/src/patient-registration/patient-registration-hooks.ts

```diff
@@ -180,49 +181,73 @@ export function useInitialFormValues(patientUuid: string): [FormValues, Dispatch
   return [initialFormValues, setInitialFormValues];
 }
 
-export function useInitialAddressFieldValues(patientUuid: string, fallback = {}): [object, Dispatch<object>] {
-  const { isLoading, patient } = usePatient(patientUuid);
+export function useInitialAddressFieldValues(
+  fallback = {},
```

Comment:
This should probably be typed? I.e., surely we know something more about the structure here than that it's an object.